roles/StackStorm.nodejs/tasks/nodejs_yum.yml

```diff
@@ -25,6 +35,7 @@
     # TODO: Allow yum downgrade since Ansible 2.4
     # https://github.com/ansible/ansible/pull/21516
     # allow_downgrade: yes
+    update_cache: "{{ rm_nodesource_repo is changed }}"
```

Comment:
Is this rm_nodssource_repo variable still defined/used?

Reply:
good catch. I should probably register a var on the previous task to do the same thing here.